package.json

```diff
@@ -124,5 +87,46 @@
         }
       ]
     ]
+  },
+  "devDependencies": {
+    "babel-cli": "^6.24.0",
+    "babel-core": "^6.24.0",
+    "babel-eslint": "^7.2.1",
+    "babel-loader": "^6.4.1",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.3.2",
+    "babel-preset-flow": "^6.23.0",
+    "babel-preset-react": "^6.23.0",
+    "babel-preset-react-hmre": "^1.1.1",
+    "browserify": "^13.0.1",
+    "chai": "^3.5.0",
+    "css-loader": "^0.26.0",
+    "eslint": "^3.10.2",
+    "eslint-config-google": "^0.6.0",
+    "eslint-plugin-flowtype": "^2.30.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-react": "^6.4.0",
+    "devtools-license-check": "^0.1.0",
```

Comment:
This came up in debugger.html too - I wonder why I don't see this (and, why I did see it for devtools-core).  Anyway, easy to fix, and sorry about that.